stripe/src/main/java/com/stripe/android/paymentsheet/ui/BillingAddressView.kt

```diff
@@ -0,0 +1,133 @@
+package com.stripe.android.paymentsheet.ui
+
+import android.content.Context
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.widget.AdapterView
+import android.widget.LinearLayout
+import androidx.annotation.VisibleForTesting
+import androidx.core.content.ContextCompat
+import androidx.core.os.ConfigurationCompat
+import androidx.core.view.ViewCompat
+import com.google.android.material.shape.CornerFamily
+import com.google.android.material.shape.MaterialShapeDrawable
+import com.google.android.material.shape.ShapeAppearanceModel
+import com.stripe.android.R
+import com.stripe.android.databinding.StripeBillingAddressLayoutBinding
+import com.stripe.android.databinding.StripeCountryDropdownItemBinding
+import com.stripe.android.view.Country
+import com.stripe.android.view.CountryAdapter
+import com.stripe.android.view.CountryAutoCompleteTextViewValidator
+import com.stripe.android.view.CountryUtils
+
+internal class BillingAddressView @JvmOverloads constructor(
+    context: Context,
+    attrs: AttributeSet? = null,
+    defStyleAttr: Int = 0
+) : LinearLayout(context, attrs, defStyleAttr) {
+    private val viewBinding = StripeBillingAddressLayoutBinding.inflate(
+        LayoutInflater.from(context),
+        this
+    )
+
+    private val countryAdapter = CountryAdapter(
+        context,
+        CountryUtils.getOrderedCountries(
+            ConfigurationCompat.getLocales(context.resources.configuration)[0]
+        ),
+        R.layout.stripe_country_dropdown_item
+    ) {
+        StripeCountryDropdownItemBinding.inflate(
+            LayoutInflater.from(context),
+            it,
+            false
+        ).root
+    }
+
+    internal val countryView = viewBinding.country
```

Comment:
Can this be `@VisibleForTesting` rather than `internal`?

Reply:
Will do in a future PR!